CHANGELOG.md

```diff
@@ -8,32 +8,78 @@ core section.***
+- New radio components for forms. (PR #5846)
 
-#### Core
+#### Updates and Improvements
+- PHP minimum requirements raised to PHP 7.3 (PR #5723)
 - Menus are now maintained by modules and no longer in the SQL database (PR #5839)
-- Very old instruments relying on QuickForm may have issues due to code changes (PR #4928)
-- Unix user permissions have been updated which may affect access to files. New
-documentation for file permissions has been added to the README.md file (PR #5323)
+- New script created for streamlining instantiation of the Raisinbread demonstration 
+database (PR #5260)
+- New documentation for file permissions has been added to the README.md file. (PR #5323)
+- Study progression performance improvement (PR #5887)
 
-#### Modules 
+#### Bug Fixes
+- Fix edge-case that gave a confusing error message when changing password (PR #5956)
+- Fix bug where examiner form field was incorrectly hidden (PR #6085)
 
-##### Issue Tracker
-- The issue_tracker module now has the feature of uploading attachments to new or existing issues.
+### Modules 
 
-##### Battery Manager
- - New module created to manage the entries in the test_battery table of the database.
- This allows projects to modify their instrument battery without requiring backend access.
+#### Battery Manager
+- New module created to manage the entries in the test_battery table of the database.
+This allows projects to modify their instrument battery without requiring backend access.
  (PR #4221)
+ 
+#### Issue Tracker
+- The issue_tracker module now has the feature of uploading attachments to new or existing
+issues. (PR #5394)
+ 
+#### Module Manager
+- New module created to manage the status of installed modules. (PR #6015)
 
-##### Module Manager
- - New module created to manage the status of installed modules (PR #6015)
+#### Electrophysiology Browser
+- New module created to view electrophysiology data within LORIS. (PR #5230)
 
-##### Electrophysiology Browser
- - New module created to view electrophysiology data within LORIS. (PR #5230)
+#### Candidate Parameters
+- Add tab to view and edit a candidate's date of birth. (PR #4915)
+- Add date of death feature for LORIS. Store and calculate age based on a date of 
+death for candidates. (PR #4929)
 
+#### Create Timepoint
+- Add language parameter when creating a new timepoint for multilingual studies. (PR #4976)
+ 
+#### Data Release
+- UI improvements, bugfixes and addition of a filter for searching across release 
+files. (PR #5224)
 
-#### Clean Up
+
+### Clean Up
+- New tool for detection of multiple first visits for a candidate (prevents a database
+exception). It is recommended to run this tool for existing projects (PR #5270)
+- Heroku postdeploy script cleanup (PR #5275)
+- Vagrantfile Deprecation (PR #5319)
+- The `data_integrity_flag` module was removed. (PR #5824)
+
+### Notes For Existing Projects
+- PHP should be upgraded to 7.3 to before upgrading LORIS. (PR #5723)
```

Comment:
```suggestion
- PHP should be upgraded to 7.3 to before upgrading LORIS.
```
Small point but linking here implies that there are upgrade code instructions here, I feel.

Reply:
I dont really care

@driusan ?

Reply:
I also don't really care but see his point. It seems like it's just a statement and the PR doesn't give extra info.